src/System.IO.Pipelines/ref/System.IO.Pipelines.csproj

```diff
@@ -4,7 +4,7 @@
     <Configurations>netstandard-Debug;netstandard-Release</Configurations>
     <!-- We only plan to use this ref in netcoreapp. For all other netstandard compatible frameworks we should use the lib
     asset instead. -->
-    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard'">netcoreapp2.0</PackageTargetFramework>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard'">netcoreapp3.0</PackageTargetFramework>
```

Comment:
@joperezr would this be a correct change? Were some R2R issues with this package previously for ASP.NET that were fixed with https://github.com/dotnet/corefx/pull/38731

ASP.NET now currently only targets netcoreapp3.0 and I was having packaging issues trying include a reference for `ManualResetValueTaskSourceCore` until I reached this current state. While it complies and packages, is it correct for ASP.NET's packaging?

Reply:
mm I'm not sure why you would want to do this change, even if ASP.NET is targeting netcoreapp3.0, if they reference the package they would still get the netcoreapp2.0 asset resolved, which is what you want here right? I don't have full context on this change but I don't think doing this is correct. cc: @ericstj

Reply:
Would want ASP.NET to resolve the netcoreapp3.0 asset; don't want a netcoreapp2.0 asset as it needs the extra downlevel `Microsoft.Bcl.AsyncInterfaces` assembly

Reply:
it won't be a netcoreapp2.0 asset, it will be a netstandard reference assembly and it will be the same one you will get no matter how you package it. If you really want to not depend on AsyncInterfaces, then you would need to cross compile the reference assembly for netcoreapp3.0. What this line here is doing, is simply telling our infrastructure to grab the assembly that targets netstandard2.0 and package it as if it was cross-compiled for netcoreapp2.0, but it still has the same dependencies. What you really want here, is to add a netcoreapp3.0 target on the configurations.props of this ref project so that we actually target netcoreapp3.0 and won't have the dependency. Does that make sense?

Reply:
If you prefer, I'm happy to push a change directly in your PR to fix this configuration issue and to make sure that the package is built correctly and that you won't get AsyncInterfaces dependency in ASP.NET.

Reply:
I'd love that! :)

Reply:
Done, that last changed I pushed should fix this and produce the right package.